contrib/opencensus-ext-django/README.rst

```diff
@@ -40,6 +38,30 @@ for a complete reference.
             )''',
         }
     }
+    
+Additional configuration can be provided, please read
+`Customization <https://github.com/census-instrumentation/opencensus-python#customization>`_
+for a complete reference.
+
+For customization the exporter has to be instantiated separately
```

Comment:
Technically this does not HAVE to be instantiated separately to work. The only reason why we need to pass in the instance is because we need to utilize the `add_telemetry_processor` api.

I would reword this to a more generic:

`For customization of the exporter: `

.. code:: python
    from opencensus.ext.azure.trace_exporter import AzureExporter
    exporter = AzureExporter(service_name='mysite')

    OPENCENSUS = {
        'TRACE': {
            'SAMPLER': 'opencensus.trace.samplers.ProbabilitySampler(rate=1)',
            'EXPORTER': exporter
        }
    }

Reply:
Not exactly sure what you are looking for here.  I left the line whit `add_telemetry_processor` because that is the part I was trying to explain.